### ISSUE TITLE
Win11 22H2 compatibility

### ISSUE THREAD

**Issue by mikolajek** [tarcode-apps/BatteryMode#73]
The current build doesn't hide the default Windows battery icon when BM is used. Can we have this fixed, please?

**Comment by bcutter** [comment]
One year after... doesn't look like this is gonna happen, right? :-(

https://github.com/tarcode-apps/BatteryMode/issues/78 probably the same.

**Comment by bcutter** [comment]
Well, doing some quick research it seems like it's technically impossible at all. So I guess this great application won't be able to hide it too.

**Comment by ZPNRG** [comment]
Yes, as bcutter said, it appears that the Battery icon along with the Wi-Fi/networking and Sound/Volume icons are considered critical system icons/components and cannot be removed. This is still true as of 24H2. At least Microsoft is not providing a way to do so that is trivial. It is unclear if there is a registry hack or other means to do so.

**Comment by mbloflin** [comment]
I am running into the exact same issue on a 24/7 docked laptop. The native Windows 11 "fused" icon cluster makes the registry hack (HideSCAPower) obsolete. 

Since the registry is no longer a viable path, would it be possible for Battery Mode to implement a "UI Toggle" that utilizes a shell hook (similar to how Windhawk mods function) to suppress the native battery icon? Having the clean "Percentage Only" display from Battery Mode is exactly what a minimalist setup needs, but having two icons side-by-side defeats the purpose. +1 for this feature!
